confero-api/src/test/java/org/zpi/conferoapi/application/ApplicationControllerTest.java

```diff
@@ -0,0 +1,91 @@
+package org.zpi.conferoapi.application;
+
+import io.restassured.RestAssured;
+import org.junit.jupiter.api.Test;
+import org.openapitools.model.*;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.http.HttpStatus;
+import org.zpi.conferoapi.IntegrationTestBase;
+import org.zpi.conferoapi.conference.ConferenceEdition;
+import org.zpi.conferoapi.orcid.OrcidService;
+import org.zpi.conferoapi.user.User;
+
+import java.time.Instant;
+import java.util.List;
+
+import static java.time.temporal.ChronoUnit.DAYS;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.mockito.Mockito.when;
+
+class ApplicationControllerTest extends IntegrationTestBase {
+
+    private static final String ORCID = "0000-0002-5678-1234";
+    private static final String EMAIL = "example@gmail.com";
+
+    @MockBean
+    OrcidService orcidService;
+
+    @Test
+    void createApplicationSuccess() {
+        userRepository.save(User.builder()
+                .email(EMAIL).isAdmin(false).build());
+
+        conferenceEditionRepository.save(ConferenceEdition.builder()
+                .id(1L)
+                .createdAt(Instant.now())
+                .applicationDeadlineTime(Instant.now().plus(2, DAYS))
+                .build());
+
+        when(orcidService.getRecord(ORCID)).thenReturn(new OrcidInfoResponse().name("John").surname("Doe"));
+
+
+
+
+        // Prepare mock data for CreateApplicationRequest
+        var presentationRequest = new PresentationRequest()
+                .title("Introduction to AI")
+                .addPresentersItem(new PresenterRequest()
+                        .email("presenter1@example.com")
+                        .isSpeaker(true)
+                        .orcid(ORCID));
+
```

Comment:
I would include additional tests beyond the happy path, such as attempting to create an application when there is no active conference edition, among others.